nano/node/bootstrap/bootstrap.cpp

```diff
@@ -865,10 +873,27 @@ bool nano::bootstrap_attempt::lazy_finished ()
 	return result;
 }
 
+bool nano::bootstrap_attempt::lazy_has_expired ()
```

Comment:
can this be "method const"? `bool nano::bootstrap_attempt::lazy_has_expired () const`